packages/code-du-travail-frontend/pages/accessibility.js

```diff
@@ -0,0 +1,138 @@
+import {
+  Container,
+  PageTitle,
+  Section,
+  theme,
+  Title,
+  Wrapper,
+} from "@socialgouv/cdtn-ui";
+import React from "react";
+
+import { FocusRoot } from "../src/a11y";
+import Metas from "../src/common/Metas";
+import { Layout } from "../src/layout/Layout";
+
+const Accessibility = () => (
+  <Layout>
+    <Metas
+      title="Accessibilité - Code du travail numérique"
+      description="Accessibilité du site du Code du travail numérique"
+    />
+    <Section>
+      <Container narrow>
+        <FocusRoot>
+          <PageTitle>Accessibilité</PageTitle>
+        </FocusRoot>
+        <Wrapper variant="main">
+          <Title shift={theme.spacings.larger}>
+            Déclaration d’accessibilité
+          </Title>
+          <p>
+            Le Ministère du travail, de l’emploi et de l’insertion s’engage à
+            rendre son service accessible conformément à l’article 47 de la loi
+            n° 2005-102 du 11 février 2005.
+          </p>
+          <p>
+            A cette fin, il met en œuvre la stratégie et l’action suivante :
+            réalisation d’un audit de conformité à la milieu de l’année 2021.
+          </p>
+          <p>
+            Cette déclaration d’accessibilité s’applique au site Internet
+            code.travail.gouv.fr.
+          </p>
+          <Title shift={theme.spacings.larger}>État de conformité</Title>
+          <p>
+            Le site Internet code.travail.gouv.fr n’est pas encore en conformité
+            avec le référentiel général d’amélioration de l’accessibilité
+            (RGAA). Le site n’a pas encore été audité.
+          </p>
+          <p>
+            Nous tâchons de rendre dès la conception, ce site accessible à
+            toutes et à tous.
+          </p>
+          <Title shift={theme.spacings.larger}>Résultat des tests</Title>
+          <p>
+            L’audit de conformité est en attente de réalisation (milieu de
+            l’année 2021).
+          </p>
+          <Title shift={theme.spacings.larger}>
+            Établissement de cette déclaration d’accessibilité
+          </Title>
+          <p>Cette déclaration a été établie le 1 avril 2021.</p>
```

Comment:
:)

Reply:
Indeed, I did not pay attention to the date 😂